website/docs/docs/deploy/continuous-integration.md

```diff
@@ -0,0 +1,57 @@
+---
+title: "Continuous integration in dbt Cloud"
+sidebar_label: "Continuous integration"
+description: "You can set up Slim continuous integration (CI) checks to test every single change prior to deploying the code to production just like in a software development workflow."
+---
+
+To implement a continuous integration (CI) workflow in dbt Cloud, you can set up automation that tests code changes by running [Slim CI jobs](/docs/deploy/slim-ci-jobs) before merging to production. dbt Cloud tracks the state of what’s running in your production environment so, when you run a Slim CI job, only the modified data assets in your pull request (PR) and their downstream dependencies are built and tested in a staging schema. You can also view the status of the CI checks (tests) directly from within the PR; this information is posted to your Git provider as soon as a Slim CI job completes. Additionally, the teams in your organization can enable settings with your Git provider that allow PRs only with successful CI checks be approved for merging.  
+
+<Lightbox src="/img/docs/dbt-cloud/using-dbt-cloud/ci-workflow.png" width="90%" title="Workflow of continuous integration in dbt Cloud"/>
```

Comment:
This is a neat diagram! The only nit is that the yellow arrows make it seem like the pull request is being merged into the staging database. I was confused because the arrows don't go back into the main branch. It took me a minute. The red and green arrows felt similarly confusing.

Reply:
@schottj : wdyt? is this something we can update?

Reply:
I interpreted it differently. Yellow is the commit being run in the staging database for testing, before getting merged in the green production database